### ISSUE TITLE
RHS - datatab not loading

### ISSUE THREAD

**Issue by Waztom** [m2ms/fragalysis-frontend#390]
Using @tibor-postek-m2ms app -  https://fragalysis-tibor-default.xchem-dev.diamond.ac.uk/viewer/react/preview/target/Mpro

Extra data tab not appearing on the RHS after uploading a dataset:

1. Upload one RHS dataset - can see data tab on RHS
2. Upload second RHS dataset - can not see data tab on RHS

Suggest adding an arrow to load the next dataset 

![RHS_datatab](https://user-images.githubusercontent.com/24798762/85730896-f0b52b80-b6f1-11ea-9d93-8ddcdcc31d7b.png)

**Comment by Waztom** [comment]
A drop-down button and list would be more user friendly 

![RHS_dat_dropdown](https://user-images.githubusercontent.com/24798762/85738904-4b518600-b6f8-11ea-9ebd-12f2fb8d7597.png)

**Comment by tibor-postek-m2ms** [comment]
merged in `rhs-filter` branch

**Comment by phraenquex** [comment]
Currently, when you move from one tab to the other (and back again), it loses the scroll status of each tab:  every time you get back to it, it's back at the top, with only as many loaded as fit on the screen (i.e. what you see first time you open the tab).

What I'd expect is that it remembers the scroll status:  how many compounds were loaded, and where the scroll-bar had been.

If this is an easy fix, fix it under this ticket;  if it's a lot of work, create a new ticket and put it bottom of "Current milesstone" swimlane, so we can discuss and prioritise.

**Comment by teriraj** [comment]
The drop-down button works good.
However, when I open new target from menu, the 3rd tab on RHS is empty. It is filled in only after selecting a tab value from the drop-down.
video: https://www.loom.com/share/45f8ae1a5eff42aca9b3586aa74c5317
